app/controllers/accounts/passwords_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Accounts::PasswordsController < Devise::PasswordsController
+  # GET /resource/password/new
```

Comment:
What is this file doing?

Reply:
All the files under controller/accounts/ are generated by devise , these files are provided in order to add any extra functionality to devise SignIn/SignUp Flow. These files stays commented unless we want to add any user desired functionality in these controllers as they are inherited from the Devise Controllers.
These controllers were generated using
rails generate devise:controllers accounts